Gopkg.lock

```diff
@@ -256,6 +256,12 @@
   packages = ["."]
   revision = "417edcfd99a4d472c262e58f22b4bfe97580f03e"
 
+[[projects]]
+  name = "github.com/moby/moby"
```

Comment:
Please replace all `github.com/moby/moby` imports to `github.com/docker/docker` for the time being. Both point to the same repo, so there's no need to vendor it twice, and that won't allow sharing objects between classes that import via one path and the others.

For now, the docker repo situation is a mess (latest versions are not tagged), which is why we pin an old version. They have an open issue to fix that in 2018 (by properly tagging releases on a `docker-ce` repo), so we'll migrate to the final git path at that time

Reply:
That's correct, keeping both will actually cause several issues we would rather avoid.